<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblLearningRate">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 16pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Î·</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumEpocas">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 16pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Num. Epocas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLearning">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEpocas">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>110</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumEpoca">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>190</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bttnEntrenar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Entrenar</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>751</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblEpocaFinal">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Epoca Final&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblencontrado">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineErrorDeseado">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>140</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblErrorDeseado">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 16pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Error Deseado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbSeno">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seno</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbCoseno">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Coseno</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbFuncion3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>((x-2)(2x+1))/(1x^2)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblK">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 16pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>k</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineK">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bttnInicializar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Inicializar</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_gaussianas" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>751</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_error" native="true">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>270</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbFuncion4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2sin(x)+cos(3x)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbFuncion5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>550</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>sin(2x) + ln(x^2)</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
